pkg/chart/chart_test.go

```diff
@@ -420,15 +424,13 @@ func (f *fakeHTTPClient) Do(h *http.Request) (*http.Response, error) {
 			// Return fake chart index (not customizable per repo)
 			body, err := json.Marshal(*f.index)
 			if err != nil {
-				fmt.Printf("Error! %v", err)
+				return nil, err
 			}
 			return &http.Response{StatusCode: 200, Body: ioutil.NopCloser(bytes.NewReader(body))}, nil
 		}
 	}
 	for _, chartURL := range f.chartURLs {
 		if h.URL.String() == chartURL {
-			// Simulate download time
-			time.Sleep(100 * time.Millisecond)
```

Comment:
Couldn't see any reason why we'd need to slow down our unit tests like this? I don't think this fake is being used for anything other than the unit-tests?

Reply:
TBH I don't remember the reason for this.